services/gateway/shared/registerHttpApi.js

```diff
@@ -29,15 +29,13 @@ const { validate, dropEmptyProps } = require('./paramValidator');
 const logger = Logger();
 const apiMeta = [];
 
+const getMethodName = method => method.httpMethod ? method.httpMethod : 'GET';
+const dropOneSlashAtBeginning = str => str.replace(/^\//, '');
+const curlyBracketsToColon = str => str.split('{').join(':').replace(/}/g, '');
```

Comment:
Can we please move them to a single file (`services/gateway/shared/apiUtils.js`), since they're being defined as duplicates in both `registerHttpApi.js` and `registerRpcApi.js`.